requirements.txt

```diff
@@ -1,3 +1,3 @@
 assemblyline
 assemblyline-v4-service
-assemblyline_service_utilities
```

Comment:
I guess it's included by something else, but shouldn't this still be required as line 14 still uses DecoderWrapper?